settings/CiviCase.setting.php

```diff
@@ -147,6 +147,21 @@
     'description' => E::ts('This setting will only allow one active instance of each case role for all case types.'),
     'help_text' => '',
   ],
+  'civicaseLimitCcAndBCC' => [
+    'group_name' => 'CiviCRM Preferences',
+    'group' => 'core',
+    'name' => 'civicaseLimitCcAndBCC',
```

Comment:
s/civicaseLimitCcAndBCC/civicaseLimitCcAndBCc ? 
Suggested renaming for consistency to match Cc